File: 5 SEM/Системы и технологии подготовки технической и издательской документации/ЛР4/Java/Main.java
/**
 * Основной класс программы, который демонстрирует работу с фигурами.
 */
public class Main {
    public static void main(String[] args) {
        Rectangle rect = new Rectangle(5, 10);
        Circle circ = new Circle(7);

        printShapeInfo(rect);
        printShapeInfo(circ);
    }

    /**
     * Выводит информацию о фигуре.
     *
     * @param shape Объект фигуры.
     */
    public static void printShapeInfo(Shape shape) {
        System.out.println("Площадь: " + shape.area());
        System.out.println("Периметр: " + shape.perimeter());
    }
}

File: 5 SEM/Системы и технологии подготовки технической и издательской документации/ЛР4/Python/main.py
from shapes import Rectangle, Circle
from utils import print_shape_info

def main():
    """
    Основная функция программы. Создает объекты фигур и выводит их параметры.
    """
    rect = Rectangle(5, 10)
    circ = Circle(7)

    print_shape_info(rect)
    print_shape_info(circ)

if __name__ == "__main__":
    main()
